modules/installation-configuration-parameters.adoc

```diff
@@ -1284,9 +1289,77 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 
 endif::vsphere,vmc[]
 
+ifdef::alibaba[]
+//From: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L20; https://github.com/openshift/openshift-docs/pull/40651/files#r792388476
+
+[id="installation-configuration-parameters-additional-alibaba_{context}"]
+== Additional {alibaba} configuration parameters
+
+Additional {alibaba} configuration parameters are described in the following table:
+
+.Optional {alibaba} parameters
+[cols=".^2m,.^3a,^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.alibabacloud.defaultMachinePlatform.description`
+|For compute machines, the configuration used when installing on Alibaba Cloud..
+|
+
+|`platform.alibabacloud.defaultMachinePlatform.imageID`
+|For compute machines, the Image ID that should be used to create ECS instance. If set, the ImageID should belong to the same region as the cluster
```

Comment:
the image ID that should be used to create an ECS instance. If set, the image ID should belong to the same region as the cluster.